application/helpers/api_helper.php

```diff
@@ -305,57 +305,38 @@ function filter_json( $source_datajson, $dataset_array = false ) {
 }
 
 
-function filter_remote_url($url, $allowed_schemes = array('http', 'https')) {
+function filter_remote_url($url) {
     if (empty($url)) {
         return null ;
     }
     $url = filter_var($url, FILTER_VALIDATE_URL, FILTER_FLAG_PATH_REQUIRED);
+
     // ban non http/https:
+    $allowed_schemes = array('http', 'https');
     $scheme = parse_url($url, PHP_URL_SCHEME);
-    if ($url && !in_array($scheme, $allowed_schemes)) {
-        $url = false;
+    if (!in_array($scheme, $allowed_schemes)) {
+        return null;
```

Comment:
At one point the distinction between false and null was important, but maybe it was to differentiate an invalid URL from an unresolvable URL but I think https://github.com/GSA/project-open-data-dashboard/commit/32f73be10bc23c0f4ad5b9ed327b1d8b3b14ee00 eliminated the need for that distinction.

You'll want to update this line to accept falsy. https://github.com/GSA/project-open-data-dashboard/blob/e706346f1aca52fd2eda9c42e99ffadfe6881ba4/application/models/Campaign_model.php#L891

Reply:
You're right, and we're returning null everywhere in the function. Will change this.